lib/misc_funcs.sh

```diff
@@ -76,6 +76,14 @@ function load_config() {
 }
 
 
+function apply_app_relative_path() {
+  if [[ -n ${app_relative_path:+x} ]]; then
+    build_path="$build_path/$app_relative_path"
```

Comment:
Is this really all that is needed? Have you tested this thoroughly?

Reply:
I think so.
I have a project which uses this buildpack and confirmed to be working on gigalixir.